pkg/server/devfile-handler.go

```diff
@@ -63,7 +64,13 @@ func (s *Server) devfileHandler(w http.ResponseWriter, r *http.Request) {
 	devfileContentBytes := []byte(data.Devfile.DevfileContent)
 	devfileObj, _, err = devfile.ParseDevfileAndValidate(parser.ParserArgs{Data: devfileContentBytes})
 	if err != nil {
-		errMsg := fmt.Sprintf("Failed to parse devfile: %v", err)
+		errMsg := fmt.Sprintf("Failed to parse devfile:")
+		if strings.Contains(err.Error(), "schemaVersion not present in devfile") {
+			errMsg = fmt.Sprintf("%v schemaVersion not present in devfile. Only devfile 2.2.0 or above is supported. The devfile needs to have the schemaVersion set in the metadata section with a value of 2.2.0 or above.", errMsg)
```

Comment:
```suggestion
			errMsg = fmt.Sprintf("%s schemaVersion not present in devfile. Only devfile 2.2.0 or above is supported. The devfile needs to have the schemaVersion set in the metadata section with a value of 2.2.0 or above.", errMsg)
```

Since we are adding it as a string

Reply:
updated